cmd/notary/tuf.go

```diff
@@ -446,6 +453,38 @@ func importRootKey(cmd *cobra.Command, rootKey string, nRepo *notaryclient.Notar
 	return []string{}, nil
 }
 
+// importRootCert imports the base64 encrypted public certificate corresponding to the root key
+// returns empty slice if path is empty
+func importRootCert(certFilePath string) ([]data.PublicKey, error) {
+	publicKeys := make([]data.PublicKey, 0, 1)
+
+	if certFilePath == "" {
+		return publicKeys, nil
+	}
+
+	// read certificate from file
+	certFile, err := os.Open(certFilePath)
+	if err != nil {
+		return nil, fmt.Errorf("error when opening certificate file: %v", err)
+	}
+	defer certFile.Close()
+
+	certPEM, err := ioutil.ReadAll(certFile)
```

Comment:
Should we be checking this error and returning it if not `nil`?  Also, if we use `ioutil.ReadFile` here, this read and the previous open section can be merged into a single function call.